php-api-pro/problem-detail.php

```diff
@@ -32,4 +32,6 @@ public function type(): string
     }
 }
 
-$statusCode = 403;
+$statusCode = 404;
+
+var_dump(ProblemDetail::tryFrom($statusCode)->type());
```

Comment:
Close..but would need a small tweak.

If tryFrom returned null, you'd get an error you tried to call type() off of it

Reply:
Oh! yeah good catch!